wfsim/strax_interface.py

```diff
@@ -320,8 +320,12 @@ def __init__(self, config):
     strax.Option('fax_config',
                  default='https://raw.githubusercontent.com/XENONnT/private_nt_aux_files/master/sim_files/fax_config_nt.json?token=AHCU5AZMPZABYSGVRLDACR3ABAZUA'),
     strax.Option('gain_model',
-                 default=('to_pe_per_run', 'https://github.com/XENONnT/private_nt_aux_files/blob/master/sim_files/to_pe_nt.npy?raw=true'),
+                 default=('to_pe_per_run',
+                          'https://github.com/XENONnT/private_nt_aux_files/blob/master/sim_files/to_pe_nt.npy?raw=true'),
```

Comment:
This doesn't actually work right?

Reply:
Right, this is not working. Any suggestions?, I am hoping to fix this some other time when we want to bring the default setting back to work again.

Reply:
I think `('to_pe_constant', 0.01)` like https://github.com/XENONnT/straxen/pull/363/files is most transparent especially if all is overwritten with some CMT value anyway

Reply:
Okay